source/preface_third.md

```diff
@@ -1,10 +1,250 @@
 # Preface to the third edition {-}
 
+One of the pleasures of writing a new edition of someone else's book is that we
+get some freedom to say good things about the earlier editions.  We get to
+praise a version of our own book.   We will do that, in the next section.  In
+the rest of the preface, we will say what we have changed, and why, and say
+a little about where this book sits in your learning and teaching about data
+analysis and statistics.
+
+## What Simon saw
+
+The book in your hands, or on your screen, is the third edition of a book
+originally called "Resampling: the new statistics", by Julian Lincoln Simon
+[-@simon1992resampling]. You will see some history of the book in Simon's
+original preface, that follows this one.  Please do read that preface; it is an
+excellent overview of what Simon was trying to do, and why.  What Simon could
+not know, when he wrote that preface, was that the method he had discovered
+would become the foundation of a revolutionary change in the teaching of data
+analysis, that we now call "data science".
+
+Simon published the first edition in 1992, but the ideas behind the book
+started much earlier, when he noticed a major problem that is familiar to
+almost every teacher of research methods - his students had a very shallow
+understanding of statistics.  The typical response to this typical insight is
+the "cookbook" method.  We try to minimize the damage by teaching recipes for
+common situations. We apply warning labels to the recipes, in the hope
+that our students will later do the right tests, even if they have little
+understanding of what those tests mean.  Of course, this is a miserable way to
+learn, and a miserable way to teach.  Our better students see that we are not
+teaching for understanding, but rehearsal, and they suffer the most.  As you
+will soon see, this book is for the teacher as well as the student.  It may be
+no comfort, but we teachers also suffer when we have to teach this way.
+
+Because this is such a common problem, many people have asked why statistics is
+hard to teach, and hard to understand, and they have come to the same insight
+as Simon; our standard statistics use a lot of mathematics, and this
+mathematics is far beyond the level that we can reasonably teach to most of our
+students [@cobb2007introductory].  As Wallis and Roberts
+[-@wallis1957statistics] point out in early textbook, the large majority of our
+students are not fluent in mathematics, and teaching statistics by way of
+mathematics is like teaching rhetoric in a foreign language.
+
+Simon went two steps further than this, although, as you will see, those two
+steps can also be one giant leap.  If he made two steps, the first was to see
```

Comment:
Couldn't quite figure out why the distinction is made between two steps and giant leap.  Perhaps make it explicit? "these two steps are a giant leap".  "If he made two steps" again confuses.